contracts/PinningManager.sol

```diff
@@ -195,22 +194,20 @@ contract PinningManager {
             isPastCurrentEndTime,
             "PinningManager: Request already active"
         );
-
         // If request exist from past, lets have clean state. Eq. force withdraw of previous money.
         if(isPastCurrentEndTime) {
             require(offer.capacity != 0, "PinningManager: provider discontinued service");
             //NO_OVERFLOW reasoning: numberOfPeriodsDeposited always bigger or equal to numberOfPeriodsWithdrawn
             uint256 toTransfer = (request.numberOfPeriodsDeposited - request.numberOfPeriodsWithdrawn).mul(request.chosenPrice);
             request.numberOfPeriodsWithdrawn = 0;
-            request.startDate = 0;
```

Comment:
@AuHau , do you agree with me deleting this one? We are setting it anyways on line 225

Reply:
Yeah, agree. Good idea.